docs/recipes/auth0.md

```
@@ -0,0 +1,377 @@
+import ReactDOM from 'react-dom/client';
+import { Auth0Provider } from '@auth0/auth0-react';
+import { AgentuityProvider } from '@agentuity/react';
+import { App } from './App';
+import { Auth0Bridge } from './Auth0Bridge';
+
+const AUTH0_DOMAIN = import.meta.env.AGENTUITY_PUBLIC_AUTH0_DOMAIN;
+const AUTH0_CLIENT_ID = import.meta.env.AGENTUITY_PUBLIC_AUTH0_CLIENT_ID;
+const AUTH0_AUDIENCE = import.meta.env.AGENTUITY_PUBLIC_AUTH0_AUDIENCE;
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+	<React.StrictMode>
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)